docs/pages/developer-guide.md

```diff
@@ -97,39 +87,42 @@ $ cd frameworks/myframework
 
    Take a look at `src/main/java/com/mesosphere/sdk/myframework/scheduler/Main.java`.  This is the main method for your scheduler, which will be run in DC/OS via Marathon.  It reads `svc.yml`, which defines its behavior.  If you need any advanced functionality not provided by YAML, such as complex deployment plans, you will write it here.
 
-1. Build a [package](#packaging). You must run the build.sh that is within `frameworks/myframework` directory that was just generated.
+## Building a package
+
+It is best to [build packages](#packaging) from within the dcos-commons Docker container (based on `mesosphere/dcos-commons:latest`).  To run an interactive shell session in that container you should run `./test.sh -i`.  Once you're in, you can use `build.sh` to build the package.
+
+```bash
+$ ./test.sh -i
+# ./build.sh aws
+```
+
+This will deploy your framework to DC/OS as a [package](#packaging). `build.sh` creates this package and uploads it to an AWS S3 bucket that is used to make it available to a DC/OS cluster.
```

Comment:
We would provide the option for configuring the bucket here -- since our target is most likely users outside of mesosphere.